### DISCUSSION TITLE
Unicode?

### DISCUSSION THREAD

**Discussion by ChrisJefferson** [kdl-org/kdl#5] [Q&A] [answered]
There is no mention of Unicode. Hopefully all strings (and the whole file?) Is just utf8?

The only thing (I think) this makes annoying is deciding the format of unescaped names, where it might be tempting to strongly restrict them (the problem with trying to say what Unicode can be used where is the file format then has to update for every Unicode version).

**Comment by zkat** [comment, marked as answer]
I guess I forgot to write it down, but files should always be utf8, imo.

**Reply by zkat** [reply]
It's done! https://github.com/zkat/kdl/commit/e3254789d992c31df2ca8539bdd1564739e96b7c
